openapi/platform/platform.yaml

```diff
@@ -2933,12 +2933,27 @@ paths:
                 comments:
                   type: string
                   description: more description for cancellation of products
+                deactivationType:
+                  type: string
+                  description: |-
+                    `cancel` cancel indicates we should turn the product or service off at the anniversary date, or commitment date, whichever is later and till then it will be available for use.
+
+                    `immediate` immediate indicates we should turn the product or service off immediately, it will no longer be available to use and it will still be billed normally until the anniversary date, or commitment date.
+
+                    `immediateAndStopBilling` immediateAndStopBilling indicates we should turn the product or service off immediately, it will no longer be available to use and it will no longer be billed.
```

Comment:
`immediateAndStopBilling` is for only internal purposes, so we agreed not to expose this type.